.sonarcloud.properties

```diff
@@ -0,0 +1,14 @@
+sonar.sources=.
+# Can have multiple comma-separated entries
+sonar.exclusions=**/test*/**/*
+
+sonar.tests=.
+# Can have multiple comma-separated entries
+sonar.test.inclusions=**/test*/**/*
+
+# Python version (for python projects only)
+sonar.python.version=3
```

Comment:
Do we need this?

Reply:
Since there are a couple python scripts in the cht-core repo, SonarCloud does analyze them.  I was getting a warning from SonarCloud that I needed to specify a python version (to allow for better analysis).  Once I added this config, the warning went away.

Reply:
Oh I see it now... The healthcheck script. Hopefully that goes away soon and I'll try and remember to update this config then. Thanks!